### ISSUE TITLE
dis.cmp_op doc is inaccurate in Python 3.12.2 and 3.13.0a5

### ISSUE THREAD

**Issue by Christopher-Chianelli** [python/cpython#117270]
# Bug report

### Bug description:

From https://docs.python.org/3.12/library/dis.html#opcode-COMPARE_OP,
`dis.cmp_op` should be able lookup the comparison operation string from the opcode's argument.
However,  in Python 3.12.2 and  3.13.0a5, this fails with "IndexError: tuple index out of range"
```python
import dis

def my_fun(a, b):
    return a <= b

for instruction in dis.get_instructions(my_fun):
    if instruction.opname == 'COMPARE_OP':
        print(dis.cmp_op[instruction.arg])
```
In Python 3.11.8, it prints "<=" as expected.

`dis.dis` output of `my_fun` in Python 3.12.2:
```
  1           0 RESUME                   0

  2           2 LOAD_FAST                0 (a)
              4 LOAD_FAST                1 (b)
              6 COMPARE_OP              26 (<=)
             10 RETURN_VALUE
```

`dis.cmp_op` in Python 3.12.2:
```
('<', '<=', '==', '!=', '>', '>=')
```

Workaround: use `instruction.argrepr`.

### CPython versions tested on:

3.12, 3.13

### Operating systems tested on:

Linux

### Linked PRs
* gh-117272
* gh-117274

**Comment by gaogaotiantian** [comment]
You need to check the latest [docs](https://docs.python.org/3.13/library/dis.html#opcode-COMPARE_OP) for the latest Python version. `opname` is changed in 3.13 so the lookup is different. You should use `cmp_op[opname >> 5]`

**Comment by gaogaotiantian** [comment]
Yes, the docs was changed when it was changed to `[opname >> 5]`, but it was never updated when it was changed to `[opname >> 4]`

**Comment by gaogaotiantian** [comment]
This was changed in https://github.com/python/cpython/commit/6e4e14d98fe0868981f29701496d57a8223c5407 but the docs is never updated. I can fix the 3.12 docs unless you want to do it @Christopher-Chianelli ?

**Comment by Christopher-Chianelli** [comment]
I can do it

**Comment by gaogaotiantian** [comment]
Sure, thanks for that!
